workflow/executor/pns/pns.go

```diff
@@ -217,8 +200,19 @@ func (p *PNSExecutor) Kill(ctx context.Context, containerNames []string, termina
 }
 
 func (p *PNSExecutor) ListContainerNames(ctx context.Context) ([]string, error) {
+	procs, err := gops.Processes()
```

Comment:
rather than rely on the map created by `pollRootProcesses` we can actually just look at processes